### PR TITLE
Renamed variables. Cleanup of useless code.

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: simphony_ui/couple_CFDDEM.py [hunk 1]
```diff
@@ -25,38 +24,32 @@
...
     mesh_type = "block"
 
     # Defining the wrapper for OpenFoam
-    if mode_OF == "internal":
-        wrapper_OF = openfoam_internal.Wrapper()
-        CUBAExt_OF = openfoam_internal.CUBAExt
-    elif mode_OF == "io":
-        wrapper_OF = openfoam_file_io.Wrapper()
-        CUBAExt_OF = openfoam_file_io.CUBAExt
+    if mode_cfd == "internal":
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: This is more like openfoam_params_mode. start partitioning it out in "model class"-like things, so making it a class it's just changing from `openfoam_params_mode` to `openfoam_params.mode`

File: simphony_ui/couple_CFDDEM.py [hunk 2]
```diff
@@ -25,38 +24,32 @@
...
+        cfd_wrapper = openfoam_internal.Wrapper()
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: same here. this is all openfoam stuff.

File: simphony_ui/couple_CFDDEM.py [hunk 3]
```diff
@@ -25,38 +24,32 @@
...
     # General settings
     number_iterations = 10
 
     force_type = "Stokes"
 
     # OF settings
-    num_timesteps_OF = 10
-    timestep_OF = 2.0e-4
-    visco_OF = 1.0e-3
+    num_timesteps_cfd = 10
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: openfoam again. We want to be clear that the application uses openfoam and ligghts. However, OF as an acronym is not very meaningful and immediate, so expanding it is a good idea.

File: simphony_ui/couple_CFDDEM.py [hunk 4]
```diff
@@ -127,29 +118,16 @@ def run_calc(output_path, mesh_name):
                              (chansize[0], chansize[1], chansize[2]),
                              (0.0, chansize[1], chansize[2])]
 
-            openfoam_file_io.create_quad_mesh(tempfile.mkdtemp(), mesh_name,
-                                              wrapper_OF, corner_points,
+            openfoam_file_io.create_quad_mesh(output_path, mesh_name,
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: Strange. There might be a reason why it uses mkdtemp here, like a scratch directory. You need to check if this change is actually ok or might introduce problems (e.g. conflicts in filenames, large files in an inappropriate place)

File: simphony_ui/couple_CFDDEM.py [hunk 5]
```diff
@@ -158,15 +136,7 @@ def run_calc(output_path, mesh_name):
     pc_flow.name = "flow_chain"
     pc_wall.name = "wall"
 
-    num_particles = sum(1 for _ in pc_flow.iter_particles())
-    num_particles_wall = sum(1 for _ in pc_wall.iter_particles())
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: I think we want the number of particles on the walls as well. Keep this line.

File: simphony_ui/couple_CFDDEM.py [hunk 6]
```diff
@@ -214,29 +179,25 @@ def run_calc(output_path, mesh_name):
...
         cellmat = {}
         index = {}
         gridsize = [chansize[0]/numgrid[0],
                     chansize[1]/numgrid[1],
                     chansize[2]/numgrid[2]]
 
-        for cell in mesh_wOF.iter_cells():
-            LLN = [chansize[0]*2, chansize[1]*2, chansize[2]*2]
+        for cell in mesh_cfd.iter_cells():
+            lln = [chansize[0]*2, chansize[1]*2, chansize[2]*2]
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: try to see if you get an idea of what this lln is. seems to be some sort of 3d length. A more meaningful varname would be better, but I have no clue either. See if you find any hints around,

File: simphony_ui/couple_CFDDEM.py [hunk 7]
```diff
@@ -300,24 +246,24 @@ def run_calc(output_path, mesh_name):
...
                 if force_type == "Stokes":
-                    dragforce[i] = 3.0 * math.pi * visco_OF * \
+                    dragforce[i] = 3.0 * math.pi * visco_cfd * \
                         par.data[CUBA.RADIUS] * 2.0 * rel_velo[i]
                 elif force_type == "Dala":
-                    Rnumber = dens_liquid*abs(rel_velo) * \
-                        par.data[CUBA.RADIUS] * 2.0/visco_OF
-                    Cd = (0.63+4.8/math.sqrt(Rnumber))**2
-                    dragforce[i] = 0.5*Cd*math.pi * \
+                    r_number = dens_liquid*abs(rel_velo) * \
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: this looks suspiciously like Reynold number.

File: simphony_ui/couple_CFDDEM.py [hunk 8]
```diff
@@ -300,24 +246,24 @@ def run_calc(output_path, mesh_name):
             dragforce = np.zeros(3)
             for i in range(0, 3):
                 if force_type == "Stokes":
-                    dragforce[i] = 3.0 * math.pi * visco_OF * \
+                    dragforce[i] = 3.0 * math.pi * visco_cfd * \
```
Comment (stefanoborini) [inline simphony_ui/couple_CFDDEM.py]: viscosity ?

**Review by stefanoborini — CHANGES_REQUESTED** [review]

**Comment by martinRenou** [comment]
Ok to merge ? I'll do the refactoring in an other PR

**Comment by stefanoborini** [comment]
Let me check

**Review by stefanoborini — APPROVED** [review]

**Pull request merged**